Documentation/Cassette/README.md

```diff
@@ -8,16 +8,23 @@
 | 1   | mg 996r servo         |
 
 ## Respooler
-| Qty | Item                  |
-|-----|-----------------------|
-| 4   | 608 bearings          |
-| 2   | 80mmx8mm shaft        |
-| 1   | 8x12mm one way bearing|
-| 2   | #110 Orings 13/16" ID 1 1/16" OD  |
+| Qty | Item                                                                                           |
+|-----|------------------------------------------------------------------------------------------------|
+| 5   | 608 bearings                                                                                   |
+| 2   | 80mmx8mm shaft                                                                                 |
+| 1   | HF081412 one way bearing (8mm ID, 14mm OD, 12mm L)                                             |
+| 2   | #211 Orings (13/16" ID, 1 1/16" OD)                                                            |
+| 8   | M3 Heatset inserts (5mm OD, 4mm L)                                                             |
+| 1   | ECAS04 (remove rubber spacer)                                                                  |
+| 1   | Spring (6mm OD, 1mm Wire Size, 7.5mm Compressed Length, 15mm Free Length, 37.2N Load Capacity) |
```

Comment:
I just copied this from filamentalist. It's what i had used since I had it on hand because I was gearing up to try filamentalist. It might be too long? It seems to deform the tension arm a little. Though we can probably tweak the tension arm to add a lip or something to catch on.

![PXL_20240524_151304312](https://github.com/ArmoredTurtle/8-Track-Raven-Alpha/assets/203336/06a83d41-4078-43be-9ab9-bf6ad1cd60c0)